src/altinn-app-frontend/src/components/summary/SummaryComponent.tsx

```diff
@@ -14,13 +15,18 @@ import {
   getDisplayFormDataForComponent,
 } from 'src/utils/formComponentUtils';
 import { getTextFromAppOrDefault } from 'src/utils/textResource';
-import type { IGrid, ILayoutComponent } from 'src/features/form/layout';
+import type {
+  IGrid,
+  ILayoutComponent,
+  SummaryDisplayProperties,
+} from 'src/features/form/layout';
 import type { IComponentValidations, IRuntimeState } from 'src/types';
 
 export interface ISummaryComponent {
   id: string;
   componentRef?: string;
   pageRef?: string;
+  display?: SummaryDisplayProperties;
```

Comment:
Same here. There are probably also some other options here not yet documented in the layout jsonschema..? :thinking: